src/test/java/com/bags/simulacrum/Weapon/WeaponModifierTest.java

```diff
@@ -19,6 +20,9 @@
 
 public class WeaponModifierTest {
 
+    @Spy
```

Comment:
As you were not mocking this I am assuming you are going for an integration test where this dependencies logic actually needs to execute.

If not, switch this to a @Mock with the appropriate responses in this test class.